modules/widgets/eu-cookie-law/eu-cookie-law.js

```diff
@@ -58,6 +58,8 @@
 
 		overlay.fadeOut( 400, function() {
 			overlay.remove();
+			var widgetSection = document.querySelector(".widget.widget_eu_cookie_law_widget");
+			widgetSection.parentNode.removeChild(widgetSection);
```

Comment:
A minor nitpick but just noting that since this file uses jQuery, this would've been enough. 🙃 

```js
$( '.widget.widget_eu_cookie_law_widget' ).remove();
```